src/driver/doc/amdnpu.rst

```diff
@@ -76,12 +76,26 @@ instance of ERT. Each user channel is bound to its own dedicated mailbox.
 PCIe EP
 -------
 
-NPU is visible to the x86 as a PCIe device with 3 BARS and an MSI-X interrupt
+NPU is visible to the x86 as a PCIe device with multiple BARs and an MSI-X interrupt
```

Comment:
@sonals , I thought that you meant to change "an MSI-X interrupt vector" to "some MSI-X interrupt vectors"?

Reply:
oh, Min can update it.

Reply:
fixed